### ISSUE TITLE
getkubeversion fails

### ISSUE THREAD

**Issue by ttousai** [aquasecurity/kube-bench#58]
./kube-bench master -c 1.4.11 -v1
```shell
I1004 15:34:01.582656   20082 util.go:198] executable 'etcd' not running
I1004 15:34:01.629281   20082 util.go:198] executable 'flanneld' not running
[WARN] Kubernetes version check skipped, with error getting kubectl version
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x784201]

goroutine 1 [running]:
github.com/aquasecurity/kube-bench/cmd.runChecks(0x858e95, 0x6)
        /home/niinai/go/src/github.com/aquasecurity/kube-bench/cmd/common.go:81 +0x201
github.com/aquasecurity/kube-bench/cmd.glob..func2(0xa5b880, 0xc420017cb0, 0x0, 0x3)
        /home/niinai/go/src/github.com/aquasecurity/kube-bench/cmd/master.go:28 +0x36
github.com/spf13/cobra.(*Command).execute(0xa5b880, 0xc420017bc0, 0x3, 0x3, 0xa5b880, 0xc420017bc0)
        /home/niinai/go/src/github.com/spf13/cobra/command.go:648 +0x231
github.com/spf13/cobra.(*Command).ExecuteC(0xa5bcc0, 0x7cc000, 0x0, 0x0)
        /home/niinai/go/src/github.com/spf13/cobra/command.go:734 +0x339
github.com/spf13/cobra.(*Command).Execute(0xa5bcc0, 0xa7e600, 0x0)
        /home/niinai/go/src/github.com/spf13/cobra/command.go:693 +0x2b
github.com/aquasecurity/kube-bench/cmd.Execute()
        /home/niinai/go/src/github.com/aquasecurity/kube-bench/cmd/root.go:52 +0x9b
main.main()
        /home/niinai/go/src/github.com/aquasecurity/kube-bench/main.go:22 +0x20
```
In this case context was not set.

**Comment by lizrice** [comment]
@ttousai Is this still a problem?

**Comment by schu** [comment]
I see a related bug with latest master (`85fb818e41ccc74d80a693757f98e5a1fcc45100`).

In my cluster `kubectl version` returns:

```
Client Version: version.Info{Major:"1", Minor:"8", GitVersion:"v1.8.2", GitCommit:"bdaeafa71f6c7c04636251031f93464384d54963", GitTreeState:"clean", BuildDate:"2017-11-01T09:57:59Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"9+", GitVersion:"v1.9.0-alpha.2", GitCommit:"444a161d227565681bd2a475b4e654f4fda2b50f", GitTreeState:"clean", BuildDate:"2017-11-01T21:54:10Z", GoVersion:"go1.9.1", Compiler:"gc", Platform:"linux/amd64"}
```

That fails with

```
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/aquasecurity/kube-bench/cmd.getKubeVersion(0xc4200bd770)
	/go/src/github.com/aquasecurity/kube-bench/cmd/util.go:247 +0x5aa
       ...
```

It looks like there are two issues:

* https://github.com/aquasecurity/kube-bench/blob/85fb818e41ccc74d80a693757f98e5a1fcc45100/cmd/common.go#L64 should check if `ver` is actually set (or `getKubeVersion` error out if the version can't be determined, since it seems to be needed)
* The used regexp doesn't work for all versions (maybe use `kubectl version -o json` instead and avoid regexp parsing)

**Comment by ttousai** [comment]
@schu thanks for catching this. See #69. There is an issue with earlier versions of kubectl (eg. kubectl client 1.6.4) not supporting the `--output` option. So I have used the simpler `--short` option instead, which is widely supported and has a simple output to parse.

**Comment by schu** [comment]
@ttousai thanks, fix works for me!

**Comment by lizrice** [comment]
Fixed in #69
